bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -115,6 +115,25 @@ spec:
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
 
+          check_db_health() {
+            db="${1}"
+            if [[ ! -f "${db}" ]]; then
+                  echo "${db} does not exist, skipping health check"
+                  return
+            fi
+            echo "Checking ${db} health"
+            serverid=$(ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/server_id: *\([[:xdigit:]]\+\)/\1/p')
```

Comment:
right, a little to clever, forgot the leading and trailing whitespace in the sed.  Usually the form is `s/^.*\(foo\).*$/\1/`


`$(ovsdb-tool show-log /etc/ovn/ovnnb_db.db | sed -ne '/server_id:/s/ *server_id: *\([[:xdigit:]]\+\).*/\1/p')`



 
bad: extra leading space
```
sh-4.2# serverid=$(ovsdb-tool show-log /etc/ovn/ovnnb_db.db | sed -ne '/server_id:/s/server_id: *\([[:xdigit:]]\+\)/a\1/p')
sh-4.2# echo "[${serverid}]"
[ aac7d]
```

better
````
sh-4.2# serverid=$(ovsdb-tool show-log /etc/ovn/ovnnb_db.db | sed -ne '/server_id:/s/ *server_id: *\([[:xdigit:]]\+\).*/\1/p')
sh-4.2# echo "[${serverid}]"
[ac7d]
```